### PR TITLE
HAWQ-1289 - add hive profile example with partitioning and mixed data types

### PR DESCRIPTION
**Pull request by lisakowen**
- added the requested example
- restructured the "Partition Filtering" section a bit to better incorporate the new example

### CODE REVIEW

File: markdown/pxf/HivePXF.html.md.erb [hunk 1]
```diff
@@ -134,14 +134,30 @@ Create a Hive table to expose our sample data set.
...
 3. Perform a query on `sales_info` to verify that the data was loaded successfully:
 
     ``` sql
     hive> SELECT * FROM sales_info;
     ```
 
-In examples later in this section, you will access the `sales_info` Hive table directly via PXF. You will also insert `sales_info` data into tables of other Hive file format types, and use PXF to access those directly as well.
+### <a id="get_hdfs_file_location"></a>Determine the HDFS location of a Hive Table
+
+Should you need to identify the HDFS file location of a Hive managed table, you will reference it using its HDFS file path. You can determine a Hive table's location in HDFS using the `DESCRIBE` command, for example:
```
Comment (dyozie) [inline markdown/pxf/HivePXF.html.md.erb]: Small edit - remove "you will"

File: markdown/pxf/HivePXF.html.md.erb [hunk 2]
```diff
@@ -172,7 +188,7 @@ Hive-plug-in-specific keywords and values used in the [CREATE EXTERNAL TABLE](..
 
 ## <a id="profile_hive"></a>Hive Profile
 
-The `Hive` profile works with any Hive file format. It can access heterogenous format data in a single table where each partition may be stored as a different file format.
+The `Hive` profile works with any Hive file format. This profile can also be used to access heterogenous format data in a single table where each partition may be stored in a different file format. Refer to the [Partition Filtering](#partitionfiltering) discussion later in this topic for additional information on partitioning and the `Hive` profile.
```
Comment (dyozie) [inline markdown/pxf/HivePXF.html.md.erb]: The edit to the first sentence introduces passive voice (seems like the earlier version was better).

Comment (lisakowen) [inline markdown/pxf/HivePXF.html.md.erb]: how about "You can use this profile to access heterogenous format data in a single table where each partition may be stored in a different file format."

**Comment by lisakowen** [comment]
i have some current work that will get pushed before this and cause merge conflict.  TBD - may need to close and resubmit.

**Comment by shivzone** [comment]
Content looks accurate. At the same time, I don't see why we can't use hcatalog.default.<tbl_name> even to access hive table with heterogeneous partitions. Will ask Oleksandr Diachenko to confirm this

**Comment by lisakowen** [comment]
thanks for reviewing, @shivzone.  makes sense to include hcatalog access.  i created this example a month ago before that feature was complete.  the example currently uses a hawq/pxf external table to gather the data in the HDFS files associated with 2 hive tables of different file formats.  there is no partitioned table on the hive end at the moment.  i will try this out with the goal of including both (hcatalog and external) alternatives in the example.

**Comment by lisakowen** [comment]
updated the "partitioning with heterogeneous data" example to include hcatalog integration per your request @shivzone.

**Comment by shivzone** [comment]
thanks !

File: markdown/pxf/HivePXF.html.md.erb [hunk 3]
```diff
@@ -716,7 +716,7 @@ In this example, you will use the `Hive` profile to query a Hive table named `sa
 
 The `Hive` profile supports multiple data format types. This support enables you  to query a partitioned Hive table that may be composed of data of different formats.
 
-In this example, you will use the `Hive` profile to query a partitioned Hive external table composed of the HDFS data files associated with the `sales_info` (text format) and `sales_info_rcfile` (RC format) Hive tables you created in previous exercises. You will partition the data by year, assigning the data from `sales_info` to the year 2013, and the data from `sales_info_rcfile` to the year 2016. (Ignore at the moment the fact that the tables contain the same data.)
+In this example, you will use the `Hive` profile both directly and indirectly via PXF HCatalog integration to query a partitioned Hive external table composed of the HDFS data files associated with the `sales_info` (text format) and `sales_info_rcfile` (RC format) Hive tables you created in previous exercises. You will partition the data by year, assigning the data from `sales_info` to the year 2013, and the data from `sales_info_rcfile` to the year 2016. (Ignore at the moment the fact that the tables contain the same data.)
```
Comment (dyozie) [inline markdown/pxf/HivePXF.html.md.erb]: I'd recommend breaking up the first sentence as it's quite long - like "a partitioned Hive external table. The table is composed"

File: markdown/pxf/HivePXF.html.md.erb [hunk 4]
```diff
@@ -766,20 +766,42 @@ In this example, you will use the `Hive` profile to query a partitioned Hive ext
...
+     Prague        | Dec   |              333 |     9894.77 | 2013
+     Bangalore     | Jul   |              271 |     8320.55 | 2013
+     Beijing       | Dec   |              100 |     4248.41 | 2013
+     Prague        | Jan   |              101 |     4875.33 | 2016
+     Rome          | Mar   |               87 |     1557.39 | 2016
+     Bangalore     | May   |              317 |     8936.99 | 2016
+     ...
+    ```
+
+7. Now use the PXF `Hive` profile to create a readable HAWQ external table derived from the Hive `hive_multiformpart` external table you created in the previous steps:
```
Comment (dyozie) [inline markdown/pxf/HivePXF.html.md.erb]: Remove "Now" (just start with "Use").

**Comment by lisakowen** [comment]
PR merged; closing.

**Pull request closed without merging**